pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -119,6 +119,15 @@ def hoc_get_locale_code
   Languages.get_hoc_locale_by_unique_language(@language)
 end
 
+# hourofcode.com calls this to translate tutorial's languages attribute
+def hoc_language(lang_codes_str)
+  return '' unless lang_codes_str
+
+  # Convert language codes to array and get the translated string
+  language_codes = lang_codes_str.split(',')
+  language_codes.map {|code| hoc_s(code.downcase)}.select {|code| code}.join ", "
```

Comment:
Purely out of curiosity, is a comma-separated list the appropriate output for all of our destination locales?  (I guess we can confirm by looking at existing translations of the list.)

Reply:
This was a concern we considered. I have a lack of experience of what obstacles a comma separated list has for certain languages, so I deferred to the team (as well as [our input here](https://codedotorg.slack.com/archives/C0SUN2SSF/p1635547981001300)). The conclusion I _believe_ was to go with this format being it wasn't too detrimental.